homeassistant/components/panasonic_viera/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "panasonic_viera",
   "name": "Panasonic Viera TV",
   "documentation": "https://www.home-assistant.io/integrations/panasonic_viera",
-  "requirements": ["panasonic_viera==0.3.2", "wakeonlan==1.1.6"],
+  "requirements": ["panasonic_viera==0.3.4", "wakeonlan==1.1.6", "configurator"],
```

Comment:
`"configurator"` is rather a dependency than a requirement since it is HA internal, therefore please move it to the dependencies.